components/icon-button/style.scss

```diff
@@ -21,7 +21,7 @@
 		outline: none;
 	}
 
-	&:not( :disabled ):not( [aria-disabled="true"] ):hover {
+	&:not( :disabled ):not( [aria-disabled="true"] ):not( .is-default ):hover {
```

Comment:
Actually, now that I think about this more, I'm wondering why `icon-buttons` have different hover/active/disabled styles compared to the default styles they inherit from `button` component. Can't we just remove those overriddes? (Fine to address separately and get the fix in)

Reply:
This is because we are using aria-disabled and disabled to prevent the hover styles from being applied in some cases, like the up mover on the first block. Not sure there's a better fix.

Reply:
Yes, but why it's specific to icon buttons? Shouldn't we apply this to all buttons?

Reply:
Good point. Yes.